roles/glusterfs/tasks/main.yml

```diff
@@ -4,7 +4,7 @@
   get_url:
     url: "{{ glusterfs_repo }}"
     dest: /etc/yum.repos.d/glusterfs-epel.repo
-    sha256sum: "{{ glusterfs_repo_sha256sum }}"
```

Comment:
If `sha256sum` variable is defined then deployment fails (probably, repo's SHA256 sum changed).